vscode-antimony/src/extension.ts

```diff
@@ -214,7 +218,7 @@ async function createAnnotationDialog(context: vscode.ExtensionContext, args: an
 	}
 	vscode.commands.executeCommand('antimony.sendType', lineStr, charStr, uri).then(async (result) => {
 		const selectedType = await getResult(result);
-		const selectedItem = await multiStepInput(context, initialQuery, selectedType);
+		const selectedItem = await annotationMultiStepInput(context, initialQuery, selectedType);

```

Comment:
Changing this name seems like it's not actually related to the 'rate law' pull request?  Mostly I worry about it conflicting with the same change in a different branch.